src/mindlessgen/qm/xtb.py

```diff
@@ -36,6 +36,11 @@ def optimize(
         """
         Optimize a molecule using xtb.
         """
+        super_heavy_elements = False
+        ati_original = molecule.ati.copy()
```

Comment:
Do we need `copy()` here?

Reply:
If i don´t use the ´.copy()´  function, the variable ´ati_original´ is changed in line 43 but i need the original ´molecule.ati´ list in the end.

Reply:
Ahh, understand, probably that's just a reference to the same NumPy array in memory otherwise. If you tested it, that's fine.